building_sim_plugins/building_plugins_common/src/crowd_simulator_common.cpp

```diff
@@ -202,8 +199,8 @@ bool CrowdSimInterface::_spawn_object()
   {
     auto agent_ptr = this->_menge_handle->get_agent(i);
     agent_ptr->_external = false;
-    std::string model_name = "agent" + std::to_string(i);
-    _add_object(agent_ptr, model_name, agent_ptr->_typeName, false);
+    _add_object(agent_ptr, _internal_agents[i-external_count],
```

Comment:
Why are you reversing order here?

Reply:
Are u referring to this `_internal_agents[i-external_count]`?